nixos/modules/services/backup/borgbackup.nix

```diff
@@ -318,6 +319,30 @@ in {
             ];
           };
 
+          readWritePaths = mkOption {
+            type = with types; listOf str;
+            description = ''
+              By default, borg cannot write anywhere on the system but
+              <literal>/root/.config/borg</literal> and <literal>/root/.cache/borg</literal>.
```

Comment:
This should be something like `$HOME/.config/borg`.

Reply:
Fixed :).